recipes/kaggle/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "kaggle" %}
+{% set version = "1.5.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2b8d6bf84172a3d7c056ef9987daf49dba9a051dfa93a14d31ac5d61307cf009
+
+build:
+  number: 0
+  entry_points:
+    - kaggle = kaggle.cli:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - certifi
+    - python-dateutil
+    - python-slugify
+    - requests
+    - six >=1.10
+    - tqdm
+    - urllib3 >= 1.21.1, < 1.25
```

There should be one space before the version constraint and none inside it:
```suggestion
    - urllib3 >=1.21.1,<1.25
```